### DISCUSSION TITLE
What is the best way to color SVG?

### DISCUSSION THREAD

**Discussion by alexdi220** [microsoft/WindowsAppSDK#1919] [Q&A] [answered]
Hi, how can I color the SVG? I want to use one SVG and change it's colors depending on the state: Dark\Light theme, Hover, Pressed etc.
WinAppSDK 1.0 Desktop

**Comment by andrewleader** [comment, marked as answer]
Hi @alexdi220, if you're using XAML/WinUI 3 to display the SVG, the best place to ask that question about coloring SVGs would be the XAML repository: https://github.com/microsoft/microsoft-ui-xaml
